modules/extensions-spring-support/src/test/resources/JAVA_OBJECT.type

```diff
@@ -0,0 +1,3 @@
+%type _:Pojo = @typeId("value" : "java.lang.Object") {
```

Comment:
At least for now let's avoid coupling to this grammar. This looks simple enough to assert on code